oauthclient.yml

```diff
@@ -1,7 +1,7 @@
-apiVersion: v1
+apiVersion: oauth.openshift.io/v1
 kind: OAuthClient
 metadata:
-  name: hawtio-online-dev
+  name: hawtio-online
```

Comment:
I think this was intentional as otherwise it should overwrite what the normal install process installs as the oauth client. The one for local dev should be kept as a different client.

Reply:
Backed out. Good catch. Thx.